src/pages/nft/class/_classId/_nftId.vue

```diff
@@ -523,6 +531,7 @@ export default {
   async mounted() {
     try {
       this.lazyGetUserInfoByAddresses(this.iscnOwner);
+      this.fetchIscnOwnerNFTDisplayStateList();
```

Comment:
should fetch in server side?

Reply:
Don't want to add another blocking call in fetch just for this feature
since Google actually renders javascript, I think having the `noindex` added to meta on mount might already work